includes/class-sensei-lesson.php

```diff
@@ -4037,7 +4281,8 @@ public static function user_lesson_quiz_status_message( $lesson_id = 0, $user_id
 				$status = Sensei_Utils::sensei_user_quiz_status_message( $lesson_id, $user_id, true );
 
 				if ( ! empty( $status['message'] ) ) {
-					echo '<div class="sensei-message ' . esc_attr( $status['box_class'] ) . '">' . Sensei_Wp_Kses::wp_kses( $status['message'] ) . '</div>';
+					echo '<div class="sensei-message ' . esc_attr( $status['box_class'] ) . '">' .
+						wp_kses_post( $status['message'] ) . '</div>';
```

Comment:
Are we going to break backwards compatibility by switching from `Sensei_Wp_Kses::wp_kses` to `wp_kses_post`?

Reply:
If we're not concerned, then I'm seeing that this PR removes all usage of `Sensei_Wp_Kses::wp_kses`. Should we deprecate the class?

Reply:
`Sensei_Wp_Kses::wp_kses` adds checks for media embeds. So if someone is using the `embed`, `iframe` or `source` tags, then this will break. Despite this, I'm still inclined to make this check more restrictive.

Generally speaking, this entire PR has the potential to break backwards compatibility if third parties are adding form elements to places where we now use `wp_kses_post`.

There are still a few instances where `Sensei_Wp_Kses::wp_kses` is being called, so I don't think we can get rid of it yet. I don't particularly like / agree with the functionality in this class though, so it would be nice to revamp it at some point.